java/mykola-melnyk/src/com/zoolatech/lecture4/tasks/_7/DigitToString.java

```diff
@@ -0,0 +1,32 @@
+package com.zoolatech.lecture4.tasks._7;
+
+import java.util.Arrays;
+
+public class DigitToString {
+
+    public static String convertToWord(char digit) {
```

Comment:
I think this method should be called `digitToString`, and the next one - `convertToWord`

Reply:
Done